modules/argocd-htpasswd-sample.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/using-argocd.adoc
+
+[id="argocd-htpasswd-sample_{context}"]
+= HTPasswd OAuth provider
+
+You can use ArgoCD to deploy an HTPasswd OAuth provider. The following sample defines and ArgoCD application that you can use to log in to your cluster as the `user1` user with the password `MyPassword!`.
+
+.ArgoCD HTPassword OAuth provider sample (`oauth-htpasswd.yaml`)
+----
+---
+apiVersion: config.openshift.io/v1
+kind: OAuth
+metadata:
+  name: cluster
+  annotations:
+    argocd.argoproj.io/sync-options: Prune=false
```

Comment:
This won't be needed as pruning it's not being enabled when creating the Argo CD App.

Reply:
Pruning isn't enabled by default? Has something changed here? We definitely hit this in our testing with 1.3.x.

Reply:
If I'm not mistaken when you create an app with auto-sync you need to pass the parameter `--auto-prune` as well. But now I see auto-sync is not being used so maybe the behaviour is different, will be worth confirming.